src/user.rs

```diff
@@ -421,10 +454,16 @@ fn pick_random_action(probability_to_act: usize, channels_enabled: bool) -> Soci
             SocialAction::JoinChannel
         } else if rng.gen_ratio(1, 25) {
             SocialAction::UpdateStatus
-        } else if rng.gen_ratio(1, 3) {
+        } else if rng.gen_ratio(1, 10) {
             SocialAction::AddFriend
+        } else if channels_enabled {
+            if rng.gen_ratio(1, 5) {
```

Comment:
I think this can be inlined with the `if` in the line above and dedup the send message to DMs.

Reply:
yes u r right